styles.css

```diff
@@ -464,3 +456,227 @@ footer .contact-text {
 }
 
 .submit-contact:hover { font-weight: bold; }
+
+/* DESKTOP VERSION CSS */
+@media only screen and (min-width: 600px) {
```

Comment:
Please use media query at any point between `992px` and `1100px`
for example:
```
@media screen and (min-width: 990px) {
/*
.
.
*/
}
```

Reply:
Changes have been implemented thanks